hotel-api-persistence/src/main/java/com/hotelapp/persistence/impl/HotelRepositoryImpl.java

```diff
@@ -0,0 +1,26 @@
+package com.hotelapp.persistence.impl;
+
+import com.hotelapp.persistence.HotelRepository;
+
+import javax.inject.Inject;
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.UUID;
+
+public class HotelRepositoryImpl implements HotelRepository {
+
+    @Inject
```

Comment:
I think ```@Autowired``` works the same as ```@Inject```. I used Inject since I knew is part of the standard JSR-330, and always I was told to use it thinking on portability, so in the future it could be easier to migrate to another framework. I agree to change it if we think about following the framework standards, and that the probability to change to another framework was low.

Totally agree with Constructor Injection.

Reply:
Yup, the portability argument makes sense, but yeah most of the times, as you said, changing the dependency injection framework has a really low probability to happen. 

Let's keep the code readable and not cluttered with unnecessary annotations.